src/Projection/PdoEventStoreProjector.php

```
@@ -633,8 +638,15 @@ private function createProjection(): void
 EOT;
 
         $statement = $this->connection->prepare($sql);
-        // we ignore any occuring error here (duplicate projection)
-        $statement->execute([$this->name, $this->status->getValue()]);
+        try {
+            $statement->execute([$this->name, $this->status->getValue()]);
+        } catch (\PDOException $exception) {
```

Also let's throw a custom exception, like `ProjectionNotCreated`